README.md

```diff
@@ -74,3 +74,18 @@ Optional dependencies are available for remote storage providers ([see documenta
 
 ## Community
 Join our [community](https://discord.gg/ayj5wdAArV) on Discord to get support and leave feedback
+
+### Local install
+
+Contributors, if you want to install and test locally:
```

Comment:
Thanks for adding this,
Unfortunately those guidelines are deprecated, `make setup` and `make tests` are the new way to go!

Reply:
done